index.html

```diff
@@ -5420,6 +5420,11 @@ <h2 id="AudioWorkletNode">
             the node to be retained in memory and perform audio processing in
             the absence of any connected inputs.
           </p>
+          <dl title="interface AudioParamMap" class="idl">
+            <dt>
+              readonly maplike&lt;DOMString, AudioParam&gt;
+            </dt>
```

Comment:
That's weird. I can see it on both the remote preview and the local one. Could you please check again?